appveyor.yml

```diff
@@ -1,37 +1,41 @@
 # Based on the "trust" template v0.1.1
 # https://github.com/japaric/trust/tree/v0.1.1
 
+os: Visual Studio 2017
+
 environment:
   global:
-    RUST_VERSION: stable
     CRATE_NAME: cobalt
     APPVEYOR_CACHE_SKIP_RESTORE: true
+    MSBUILD_NODE_CONNECTION_TIMEOUT: 0
```

Comment:
What value does this give us?

Reply:
This was one of the workaround for the PDB issue, I'll try without it :)